src/entity/Bookmark.java

```diff
@@ -1,4 +1,27 @@
 package entity;
 
-public class Bookmark {
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bookmark implements BookmarkInterface {
+    /** Bookmarks that user made to save pets they are interested in adopting.
+    */
+    private Pet pet;
+    private LocalDateTime bookmarkedDate;
+
+    public Bookmark(Pet petProfile, LocalDateTime time) {
+        this.pet = petProfile;
+        this.bookmarkedDate = time;
+    }
+
+    public void createBookMark(Pet pet, LocalDateTime bookmarkedDate){
```

Comment:
I think we should still have a Bookmark class and interface because factories don't have attributes, so it doesn't store the appropriate object's properties without the class+interface